rlclientlib/logger/http_transport_client.h

```diff
@@ -89,4 +99,137 @@ namespace reinforcement_learning {
     i_trace* _trace;
     error_callback_fn* _error_callback;
   };
+
+  template <typename TAuthorization>
+  http_transport_client<TAuthorization>::http_request_task::http_request_task(
```

Comment:
Can you please place the function implementations below the class declaration (still in the .h file since its a template) so that the class functions can be read without needing to sift through the impl?

Reply:
Thanks for confirming. Resolving conversation.